addons/calico-vxlan/README.md

```diff
@@ -0,0 +1,34 @@
+# Calico addon (with VXLAN)
```

Comment:
Can you please create a copy of this document in the kubermatic/docs repo?

Reply:
https://github.com/kubermatic/docs/pull/363